torch/nested/_internal/nested_tensor.py

```diff
@@ -188,23 +222,12 @@ def _get_min_seqlen(self):
             # compute & cache
             min_val = _get_sdpa_extreme_seqlen(
                 torch.min,
-                self._offsets.diff() if self._lengths is None else self._lengths,
+                self.offsets().diff() if self._lengths is None else self._lengths,
             )
             min_seqlen_tensor = _store_val_in_tensor(min_val)
             self._metadata_cache["min_seqlen"] = min_seqlen_tensor
         return _load_val_from_tensor(min_seqlen_tensor)
 
-    # Private accessors used for treating min / max seqlen as inner tensors for
-    # flatten / unflatten. These must be properties to work with the traceable wrapper
-    # subclass logic. These do not compute / cache if not present.
-    @property
-    def _max_seqlen_tensor(self) -> Optional[torch.Tensor]:
-        return self._metadata_cache.get("max_seqlen", None)
-
-    @property
-    def _min_seqlen_tensor(self) -> Optional[torch.Tensor]:
-        return self._metadata_cache.get("min_seqlen", None)
```

Comment:
subsumed by the custom getattr I assume? My personal opinion is that properties are less disruptive

Reply:
Yeah subsumed (deferring to this thread https://github.com/pytorch/pytorch/pull/141842/files#r1887706091 for getattr/property discussion.)